pages/blog/a_straight_guide_to_apis_and_architecture_concepts.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "A straight guide to APIs and architecture concepts"
+subtitle: From a newcomer experience
+date: 2023-01-12T06:00:00+01:00
```

Comment:
please set next Tuesday as @AceTheCreator article was completed first and we will release it today or tomorrow

Reply:
Yes, of course. There is no rush 🙂